sqlmesh/dbt/basemodel.py

```diff
@@ -365,16 +377,8 @@ def _extract_value(node: t.Any) -> t.Any:
         return config
 
     def _ref(self, package_name: str, model_name: t.Optional[str] = None) -> BaseRelation:
-        if package_name in self.context.models:
-            relation = BaseRelation.create(**self.context.models[package_name].relation_info)
-        elif package_name in self.context.seeds:
-            relation = BaseRelation.create(**self.context.seeds[package_name].relation_info)
-        else:
-            raise ConfigError(
-                f"Model '{package_name}' was not found for model '{self.config.table_name}'."
-            )
         self._captured_dependencies.refs.add(package_name)
-        return relation
+        return BaseRelation.create()
```

Comment:
Do we need to provide some fake info here? Wouldn't attributes be Nones otherwise?

Reply:
What fake info are you thinking? I did confirm that our test and partner projects work with this change.

Reply:
if this works then all good